utils/bazel/llvm-project-overlay/clang-tools-extra/clangd/BUILD.bazel

```diff
@@ -0,0 +1,45 @@
+# This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])
+
+cc_library(
+    name = "clangd_library",
```

Comment:
clangDaemon would match the library name in clang-tools-extra/clangd/CMakeLists.txt

Reply:
Renamed

Reply:
Small nit: IIUC the convention in the llvm bzl tree is to use casing like `ClangDaemon`.

Reply:
Done.